src/python/grpcio/tests/interop/_insecure_interop_test.py

```diff
@@ -31,12 +31,12 @@
 
 import unittest
 
+from gen.src.proto.grpc.testing import test_pb2
```

Comment:
is there a way for this to stay `from grpc.testing import test_pb2`? I think that's actually a requirement.